services/blockchain-indexer/shared/dataService/business/pos/stakers.js

```diff
@@ -38,7 +39,8 @@ const getStakesIndex = () => getTableInstance(
 
 const getStakers = async params => {
 	const stakesTable = await getStakesIndex();
```

Comment:
Can we rename it to `getStakesTable` just to be consistent:
```suggestion
	const stakesTable = await getStakesTable();
```

Reply:
Applied.

Reply:
Applied.